utils/index.js

```diff
@@ -18,3 +19,19 @@ exports.getReactScriptsMajorVersion = function(cliVersion) {
   const major = semver.major(dependencies['react-scripts']);
   return Number(major);
 };
+
+exports.getReactScriptsPatchVersion = function(cliVersion) {
```

Comment:
We'll need to get the minor as well so I think instead of adding another function again to take care of the minor, we'd be better off keeping only one function. It would still get the cli version as a parameter but it would split the version number given and return an object with each part of that version i.e `{ major: 2, minor: 0, patch: 3 }`. That way we have only one function call to make to have everything we need. Of course we'll keep the same check there already are.